packages/nimble-components/src/rich-text/editor/tests/rich-text-editor.spec.ts

```diff
@@ -390,6 +390,165 @@ describe('RichTextEditor', () => {
             ]);
         });
 
+        describe('should render as a plain text for bold and italics input rule entered into the editor', () => {
+            const markdownInput: { name: string, input: string }[] = [
+                { name: 'bold(**)', input: '**bold**' },
+                { name: 'bold(__)', input: '__bold__' },
+                { name: 'italics(*)', input: '*italics*' },
+                { name: 'italics(_)', input: '_italics_' }
+            ];
+            const focused: string[] = [];
+            const disabled: string[] = [];
+
+            for (const value of markdownInput) {
+                const specType = getSpecTypeByNamedList(
```

Comment:
It was recently merged but you may notice that `getSpecTypeByNamedList` is deprecated.

Can you switch to `parameterizeNamedList` for this PR and for future changes? Ideally in a follow-up PR migrate existing tests to the new pattern as well. May be useful to share with others as well as new changes should not use the deprecated method.

The PR shows some example tests switching to the new pattern: https://github.com/ni/nimble/pull/1379

Some things in particular to notice:
- The array of test cases should be marked `as const`: https://github.com/ni/nimble/pull/1379/files#diff-fb05c735c3c2ee54c2eefad311e72a107fdb78ff88b242c88ac773ac0e22a080R330
- For consistency, keep the parameter names for the callback the same, i.e. `spec`, `name`, `value`: https://github.com/ni/nimble/pull/1379/files#diff-fb05c735c3c2ee54c2eefad311e72a107fdb78ff88b242c88ac773ac0e22a080R332

Reply:
Sure, thanks for letting us know and sharing the references. I have updated the newly added test cases with the new pattern and will create a separate PR to update in other test cases.